src/geo/leaflet/leaflet-map-view.js

```diff
@@ -45,8 +45,15 @@ var LeafletMapView = MapView.extend({
       // remove the "powered by leaflet"
       this.map_leaflet.attributionControl.setPrefix('');
 
+      // Disable scrollwheel
       if (this.map.get("scrollwheel") == false) this.map_leaflet.scrollWheelZoom.disable();
+      // Disable keyboard
       if (this.map.get("keyboard") == false) this.map_leaflet.keyboard.disable();
+      // Disable dragging (also doubleClickZoom)
```

Comment:
I'll change it before merge `v4` with `widgets` branch. Probably not well expressed the comment. The idea is to indicate the dragging and double click options are disabled because the `drag` options is disabled.

Another option, probably better, is not comment here and comment appropriately the `drag` option in   the map model class.